packages/api-kit/src/SafeApiKit.ts

```diff
@@ -852,6 +854,66 @@ class SafeApiKit {
       }
     })
   }
+
+  /**
+   * Returns the list of confirmations for a given a SafeOperation.
+   *
+   * @param getSafeOperationConfirmationsProps - The parameters for fetching the list of confirmations
+   * @returns The list of confirmations
+   * @throws "Invalid SafeOperation hash"
+   * @throws "Invalid data"
+   */
+  async getSafeOperationConfirmations({
```

Comment:
Please double check how we do this. This kind of looks like a getSafeOperationConfirmationsByHash with a different name as the hash itself is a part of a "perma-link". In this case, it would be common to separate the interface as `getSafeOperationConfirmations(hash, { limit, offset })` since that would be somehow associated with a domain model.

Reply:
Updated the function signature accordingly 👍